mdn/kuma
# Add Payments Thank You page

**Changes:**
- Add React Thank You page with feedback form
- Add endpoint, `/api/v1/subscriptions/feedback` that posts GA event 
- Remove Jinja FAQ page
- Update urls to point to React FAQ page instead of Jinja FAQ page
- Move relevant pages to non-wiki site (/payments/, /payments/terms/, /payments/thank-you/)
- Add React Testing Library

**To test:** 
1. Nothing links to this page yet, but you can go direcly to:
http://localhost.org:8000/en-US/payments/thank-you/

2. Type a string less than 5 characters in the Feedback form, should see this error message on submit: 
<img width="632" alt="Screen Shot 2020-03-31 at 2 03 56 AM" src="https://user-images.githubusercontent.com/650/77992363-eacacc80-72f3-11ea-8f57-8d836ac85092.png">

3. If backend request fails, should see a different error message: 
<img width="1131" alt="Screen Shot 2020-03-31 at 1 54 50 AM" src="https://user-images.githubusercontent.com/650/77992208-a2131380-72f3-11ea-9b3e-57a64cc341d6.png">

4. Type a string greater than 5 characters, submit form, should see a Thank you message:
<img width="1086" alt="Screen Shot 2020-03-31 at 1 53 18 AM" src="https://user-images.githubusercontent.com/650/77992177-8c9de980-72f3-11ea-9c47-b38ea0625a79.png">

5. On http://wiki.localhost.org:8000/en-US/profiles/test-super/edit, verify that FAQ link now goes to /payments/:
<img width="793" alt="faq" src="https://user-images.githubusercontent.com/650/77326278-f284e280-6cef-11ea-8575-19f307483a2c.png">

6. On http://wiki.localhost.org:8000/en-US/payments/recurring/management, verify that "Monthly Subscriptions FAQ" links to /payments/#contribute-monthly-faq:
<img width="523" alt="faq2" src="https://user-images.githubusercontent.com/650/77326627-7343de80-6cf0-11ea-9934-538579da5c24.png">

Closes #6581
